app/server/appsmith-git/src/main/resources/git.sh

```
@@ -0,0 +1,87 @@
+    local redis_key="$1"
+
+    echo "lock:${redis_key}"
+}
+
+# Clones git repo using SSH key
+git_clone() {
+    local private_key="$1"
+    local remote_url="$2"
+    local target_folder="$3"
+
+    local temp_private_key=$(mktemp /dev/shm/tmp.XXXXXX)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)